### ISSUE TITLE
Ligatures for Ubuntu Linux 18.04 and LibreOffice 6.3.6.2

### ISSUE THREAD

**Issue by GreatCharter** [silnrsi/font-tagmukay#3]
My question is similar to the previous Mac question only I'm using a Linux machine.
I have a PC laptop with a US physical keyboard.
My operating system is Ubuntu Linux 18.04
I'm testing Tifinagh typing in LibreOffice 6.3.6.2 with the Tagmukay font chosen
I have Keyman 13 for Linux running loaded and the Tawallammat Tifinagh (SIL) Keyboard activated in ibus and chosen in the dropdown menu.

Characters accessible from the keyboard layout for the normal keyboard and with the shift key work by characters requiring the e AltGr key don't seem to work:

- When I type AltGrm I don't get the Tifinagh end-of-sentence marker
- When I type ⵔ,AltGr+7,ⵜ (erra+AltGR7+etta) no transformation takes place.
- I also tried entering _erra_ then the unicode and pressing Alt-x before the _etta_ as was suggested in the response to the Mac question, the symbol for the consonant joiner appeared between the erra the etta, but again no transformation occurred.

**Comment by GreatCharter** [comment]
I found the solution for the AltGr key not working properly.  It was a Linux 18.04 issue regarding the AltGr key.  Normal keyboard settings Ubuntu Linux 18.04 did not allow for the needed changes, so in order to make the AltGr key work as needed, I had to do the following:
install gnome-tweaks 
open tweaks and go to the menuitem 'keyboard & mouse'
Next to the "compose key" line click the "disabled" button
In the resultant dialog click the toggle switch to turn compose key function on.
Then select the "Right Super" radio button
close the dialog and tweaks

Now while in LibreOffice writer if I type ⵔ,AltGr+7,ⵜ (erra+AltGR7+etta) the correct transformation takes place, rendering "erta" (oddly though when I do the same in this Github text editor it doesn't work, and even if I copy and paste into this text editor it comes through as ⵔ⵿ⵜ)
Also now I can easily access the end-of-sentence marker with AltGr+m to render  ⵰

Hope someone else using Linux finds this solution helpful.
